src/View/HousingPanel.java

```diff
@@ -0,0 +1,133 @@
+    * grouping the parent property belongs to.
+    * @param color the Color this HousingPanel will be set to.
+    * @throws IOException If there is a failure loading either housing
+    * image.
+    ******************************************************************/
+  public HousingPanel(Color color) throws IOException {
+    houseCounter = 0;
+
+    houseImg = ImageIO.read(new File("res/MonopolyHouse.png"));
+    hotelImg = ImageIO.read(new File("res/MonopolyHotel.png"));
+    setOpaque(true);
+    setBackground(color);
+  }
+
+  /*******************************************************************
+   * The getHouses method returns the number of houses on this panel.
+   *
+   * @return houseCounter The number houses on this panel.
+   ******************************************************************/
+  public int getHouses() {
+    return houseCounter;
+  }
+
+  /*******************************************************************
+   * The setHouses method sets the number of houses on this panel.
+   * @param houseCounter the number of houses on this panel.
+   ******************************************************************/
+  public void setHouses(int houseCounter) {
+    if(houseCounter > 5)
+      this.houseCounter = 5;
+    else if(houseCounter < 0)
+      this.houseCounter = 0;
+    else
+      this.houseCounter = houseCounter;
+    repaint();
+    revalidate();
+  }
+
+  /*******************************************************************
+   * The paintComponent draws the houses in the panel based off the
+   * current number of houses this panel currently has.
+   * @param g Graphics component of this panel.
+   ******************************************************************/
+  @Override
+  public void paintComponent(Graphics g) {
+    super.paintComponent(g);
+    switch (houseCounter) {
+      case 1:
+        g.drawImage(houseImg,10,20,40,50,null);
+        break;
+      case 2:
+        g.drawImage(houseImg,10,20,40,50,null);
+        g.drawImage(houseImg,50,20,40,50,null);
+        break;
+      case 3:
+        g.drawImage(houseImg,10,20,40,50,null);
+        g.drawImage(houseImg,50,20,40,50,null);
+        g.drawImage(houseImg,90,20,40,50,null);
+        break;
+      case 4:
+        g.drawImage(houseImg,10,20,40,50,null);
+        g.drawImage(houseImg,50,20,40,50,null);
+        g.drawImage(houseImg,90,20,40,50,null);
+        g.drawImage(houseImg,130,20,40,50,null);
+        break;
+      case 5:
+        g.drawImage(hotelImg,70,30,50,40,null);
+        break;
+      default:
+    }
+  }
+
+  /********************************************************************
+   * The getPreferredSize method is used to Lock the size of the Panel
+   * to the correct size.
+   * @return The dimensions of the housing panel.
+   *******************************************************************/
+  @Override
+  public Dimension getPreferredSize(){
+    return new Dimension(200,100);
```

Comment:
this should provably be some final variable

Reply:
Agreed